Code/Framework/AzCore/AzCore/Task/TaskGraphSystemComponent.cpp

```diff
@@ -15,8 +15,14 @@
 #include <AzCore/Component/ComponentApplicationBus.h>
 #include <AzCore/Threading/ThreadUtils.h>
 
+ // PERFORMANCE NOTE & TODO
+ // Profiling Ros Con demo, Task Graph was 2-3ms slower than Jobs
+ // Time for Jobs was ~5.5ms, maxing out at ~6.3ms
+ // Task Graph took ~7.7ms, maxing out at ~8.7ms
+ // Disabling cl_activateTaskGraph for performance reasons for the time being
+
 // Create a cvar as a central location for experimentation with switching from the Job system to TaskGraph system.
-AZ_CVAR(bool, cl_activateTaskGraph, true, nullptr, AZ::ConsoleFunctorFlags::Null, "Flag clients of TaskGraph to switch between jobs/taskgraph (Note does not disable task graph system)");
+AZ_CVAR(bool, cl_activateTaskGraph, false, nullptr, AZ::ConsoleFunctorFlags::Null, "Flag clients of TaskGraph to switch between jobs/taskgraph (Note does not disable task graph system)");
```

Comment:
Shouldn't this only be set to` false` if this PR is targeting the ros con demo branch, not O3DE development?

Reply:
There are no competing metrics at the moment to suggest this optimization wouldn't also bring performance benefits elsewhere. Full frame time when toggling off/on goes from 19.2ms for Jobs up to 21.4ms for Task Grpah

Reply:
Well did you consult with @rgba16f about since they originally set the CVar to [true](https://github.com/aws-lumberyard-dev/o3de/commit/c9b814d4f96dece3dc78fa108feb10bced30c8f1)?